From bca3fc223e1a655ad15c5d09930e5e8aaaa21c3b Mon Sep 17 00:00:00 2001
From: Ryan Volz <ryan.volz@gmail.com>
Date: Sun, 13 Sep 2020 19:15:00 -0400
Subject: [PATCH] cmake: Set VERSION_PATH to be only "git", fixing
 RC_MAINT_VERSION.

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index dc253a144..be2e6d53f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -41,7 +41,7 @@ GR_REGISTER_COMPONENT("testing-support" ENABLE_TESTING)
 SET(VERSION_MAJOR 3)
 SET(VERSION_API   9)
 SET(VERSION_ABI   0)
-SET(VERSION_PATCH 0-git)
+SET(VERSION_PATCH git)
 include(GrVersion) #setup version info
 
 # Minimum dependency versions for central dependencies:
-- 
2.20.1

