From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ryan Volz <ryan.volz@gmail.com>
Date: Fri, 10 Jun 2022 13:47:41 -0400
Subject: [PATCH] qtgui: Add missing <deque> include to spectrumdisplayform.h.

Signed-off-by: Ryan Volz <ryan.volz@gmail.com>
---
 gr-qtgui/include/gnuradio/qtgui/spectrumdisplayform.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gr-qtgui/include/gnuradio/qtgui/spectrumdisplayform.h b/gr-qtgui/include/gnuradio/qtgui/spectrumdisplayform.h
index d23baf0b7..e0b58e5aa 100644
--- a/gr-qtgui/include/gnuradio/qtgui/spectrumdisplayform.h
+++ b/gr-qtgui/include/gnuradio/qtgui/spectrumdisplayform.h
@@ -23,6 +23,7 @@ class SpectrumGUIClass;
 #include <gnuradio/qtgui/WaterfallDisplayPlot.h>
 #include <QTimer>
 #include <QValidator>
+#include <deque>
 #include <vector>
 
 class SpectrumDisplayForm : public QWidget, public Ui::SpectrumDisplayForm
-- 
2.35.1

