From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jeff Long <willcode4@gmail.com>
Date: Mon, 23 Jan 2023 06:44:16 -0500
Subject: [PATCH] iio: cmake: revert libad9361 linkage back to public

Signed-off-by: Jeff Long <willcode4@gmail.com>
---
 gr-iio/lib/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gr-iio/lib/CMakeLists.txt b/gr-iio/lib/CMakeLists.txt
index facb00fe0..50c9431e0 100644
--- a/gr-iio/lib/CMakeLists.txt
+++ b/gr-iio/lib/CMakeLists.txt
@@ -31,7 +31,7 @@ if(libad9361_SUFFICIENT)
     target_sources(gnuradio-iio PRIVATE fmcomms2_sink_impl.cc fmcomms2_source_impl.cc
                                         pluto_utils.cc)
     target_link_libraries(gnuradio-iio PRIVATE libad9361::ad9361)
-    target_compile_definitions(gnuradio-iio PRIVATE -DGR_IIO_LIBAD9361)
+    target_compile_definitions(gnuradio-iio PUBLIC -DGR_IIO_LIBAD9361)
 endif(libad9361_SUFFICIENT)
 
 if(ENABLE_COMMON_PCH)
-- 
2.38.1

